components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -15,28 +14,28 @@ import {DispatchFunc} from 'mattermost-redux/types/actions';
 import {PreferenceType} from 'mattermost-redux/types/preferences';
 
 import {pageVisited, trackEvent} from 'actions/telemetry_actions';
-import {openModal} from 'actions/views/modals';
-import AlertBanner from 'components/alert_banner';
-import BlockableLink from 'components/admin_console/blockable_link';
-import FormattedMarkdownMessage from 'components/formatted_markdown_message';
-import PurchaseModal from 'components/purchase_modal';
 import FormattedAdminHeader from 'components/widgets/admin_console/formatted_admin_header';
 import {getCloudContactUsLink, InquiryType, InquiryIssue} from 'selectors/cloud';
 import {GlobalState} from 'types/store';
 import {
     Preferences,
     CloudBanners,
-    ModalIdentifiers,
     TELEMETRY_CATEGORIES,
 } from 'utils/constants';
 import {isCustomerCardExpired} from 'utils/cloud_utils';
-
-import privateCloudImage from 'images/private-cloud-image.svg';
-import upgradeMattermostCloudImage from 'images/upgrade-mattermost-cloud-image.svg';
+import {getRemainingDaysFromFutureTimestamp} from 'utils/utils.jsx';
 
 import BillingSummary from './billing_summary';
 import PlanDetails from './plan_details';
 
+import {
+    contactSalesCard,
+    cancelSubscription,
+    infoBanner,
+    creditCardExpiredBanner,
+    paymentFailedBanner,
+} from './billing_subscriptions_jsx_pieces';
```

Comment:
Given that all of these are in the `components` folder I think that implies that these are components so that word is unnecessary.

Don't like jsx in the name

components/admin_console/billing/billing_subscriptions

would be fine by me

Reply:
> components/admin_console/billing/billing_subscriptions

@nickmisasi That's already the name of the file for the parent component (the file we are commenting on right now) :)

Reply:
0/5 on which option we pick @pablovelezvidal, also open to more if you have others in mind but consensus is to not include `jsx` in the name of the file to avoid confusion

Reply:
@nickmisasi @marianunez I went for creating subfolders and place the three related files there, so one would be index.ts, the other would be i.e billing_summary.tsx and the other related one is the billing_summary.scss. That way we keep consistency.

Reply:
Looks great with the new folder organization 👍